sycl/doc/extensions/experimental/sycl_ext_oneapi_graph.asciidoc

```diff
@@ -1652,6 +1670,22 @@ block size.
 the finalize call either extending the basic command graph proposal
 or layered as a separate extension proposal.
 
+== Non-implemented features and known issues
+The following features are not yet supported:
+
+. Using `handler::fill` in a graph node implemented for USM only.
+. Using `handler::memset` in a graph node.
+. Using `handler::prefetch` in a graph node.
+. Using `handler::memadvise` in a graph node.
+. Using specialization constants in a graph node.
+. Using reductions in a graph node.
+. Using sycl streams in a graph node.
```

Comment:
Yes streams are unsupported in both modes, and an exception will be thrown when a user tries to use them in a graph node.

For all the items in this list, an exception will be thrown at point of use, rather than UB or silently failing.